src/main/charts/bitbucket/values.yaml

```diff
@@ -104,27 +104,32 @@ bitbucket:
 
   resources:
     jvm:
-      # -- The maximum amount of heap memory that will be used by the Bitbucket JVM. The same value will be used by the ElasticSearch JVM.
+      # -- JVM memory arguments below are based on the defaults defined for the Bitbucket docker container, see:
+      # https://bitbucket.org/atlassian-docker/docker-atlassian-bitbucket-server/src/master/
+      #
+      # -- The maximum amount of heap memory that will be used by the Bitbucket JVM. The same value will be used by the Elasticsearch JVM.
       maxHeap: "1g"
-      # -- The minimum amount of heap memory that will be used by the Bitbucket JVM. The same value will be used by the ElasticSearch JVM.
-      minHeap: "1g"
+      # -- The minimum amount of heap memory that will be used by the Bitbucket JVM. The same value will be used by the Elasticsearch JVM.
+      minHeap: "512m"
 
     # -- Specifies the standard Kubernetes resource requests and/or limits for the Bitbucket container.
     # It is important that if the memory resources are specified here, they must allow for the size of the Bitbucket JVM.
     # That means the maximum heap size, the reserved code cache size, plus other JVM overheads, must be accommodated.
-    # Allowing for (maxHeap+codeCache)*1.5 would be an example.
-    container: {}
+    # Allowing for maxHeap * 1.5 would be an example.
+    container:
     #  limits:
-    #    cpu: "4"
-    #    memory: "2G"
-    #  requests:
-    #    cpu: "4"
+    #    cpu: "1"
     #    memory: "2G"
+      requests:
+        # -- If changing the cpu value update additional JVM arg 'ActiveProcessorCount' below
+        cpu: "1"
+        memory: "2G"
 
   # -- Specifies a list of additional arguments that can be passed to the Bitbucket JVM, e.g. system properties
   additionalJvmArgs:
-  #    - -Dfoo=bar
-  #    - -Dfruit=lemon
+    # -- The value defined for ActiveProcessorCount should correspond to that provided for 'container.requests.cpu'
```

Comment:
@tan-ro this is something we should mention in proper documentation as well.

Reply:
Good point!